Content.Shared/GameObjects/Components/Observer/GhostWarpToTargetRequestMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components.Observer
+{
+    [Serializable, NetSerializable]
+    public class GhostWarpToTargetRequestMessage : ComponentMessage
+    {
+        public GhostWarpToTargetRequestMessage(EntityUid target)
+        {
+            Target = target;
+            Directed = true;
+        }
+
+        public EntityUid Target { get; }
```

Comment:
Same thing.

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3310/commits/712431c27a251123feeed82cf066580a24dce43e